src/snowflake/snowpark/udf.py

```diff
@@ -66,21 +70,32 @@ class UserDefinedFunction:
         - :func:`~snowflake.snowpark.functions.udf`
     """
 
+    # def __init__(
```

Comment:
Temporarily commented to show the usefulness of the `CallableProperties`. Once that implementation is accepted, this old `__init__` will be removed in favor of the new one.